### ISSUE TITLE
Panic in `glooctl check`

### ISSUE THREAD

**Issue by lgadban** [solo-io/gloo#3391]
**Describe the bug**
Seg fault in `glooctl check` 

**To Reproduce**
1. Install `glooctl` and gloo OS `1.5.0-beta10`
2. Create any `VirtualService`
3. Run `glooctl check`:
```
% glooctl check
Checking deployments... OK
Checking pods... OK
Checking upstreams... OK
Checking upstream groups... OK
Checking auth configs... OK
Checking rate limit configs... OK
Checking secrets... OK
Checking virtual services... panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x36b246e]

goroutine 1 [running]:
github.com/solo-io/gloo/projects/gloo/cli/pkg/cmd/check.renderRef(0x0, 0x0, 0x0)
	/workspace/gloo/projects/gloo/cli/pkg/cmd/check/root.go:551 +0x2e
github.com/solo-io/gloo/projects/gloo/cli/pkg/cmd/check.checkVirtualServices.func1(0x0, 0x0, 0x0, 0x0, 0x0)
	/workspace/gloo/projects/gloo/cli/pkg/cmd/check/root.go:423 +0x57
github.com/solo-io/gloo/projects/gloo/cli/pkg/cmd/check.checkVirtualServices(0xc0013f8400, 0x5, 0x8, 0xc000ccb500, 0xb, 0x10, 0x0, 0x0, 0x0, 0x0, ...)
	/workspace/gloo/projects/gloo/cli/pkg/cmd/check/root.go:432 +0xd71
github.com/solo-io/gloo/projects/gloo/cli/pkg/cmd/check.CheckResources(0xc000720000, 0x0, 0x0, 0x0)
	/workspace/gloo/projects/gloo/cli/pkg/cmd/check/root.go:112 +0xb15
github.com/solo-io/gloo/projects/gloo/cli/pkg/cmd/check.RootCmd.func1(0xc000748000, 0x5aebd08, 0x0, 0x0, 0x0, 0x0)
	/workspace/gloo/projects/gloo/cli/pkg/cmd/check/root.go:43 +0x5c
github.com/spf13/cobra.(*Command).execute(0xc000748000, 0x5aebd08, 0x0, 0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0xb21
github.com/spf13/cobra.(*Command).ExecuteC(0xc000c66000, 0xc000748000, 0x0, 0x0)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x698
github.com/spf13/cobra.(*Command).Execute(0xc000c66000, 0x0, 0x0)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864 +0x3b
main.main()
	/workspace/gloo/projects/gloo/cli/cmd/main.go:11 +0x34
```

**Additional context**
```
% glooctl version
Client: {"version":"1.5.0-beta10"}
Server: {"type":"Gateway","kubernetes":{"containers":[{"Tag":"1.5.0-beta10","Name":"discovery","Registry":"quay.io/solo-io"},{"Tag":"1.5.0-beta10","Name":"gateway","Registry":"quay.io/solo-io"},{"Tag":"1.5.0-beta10","Name":"gloo-envoy-wrapper","Registry":"quay.io/solo-io"},{"Tag":"1.5.0-beta10","Name":"gloo","Registry":"quay.io/solo-io"}],"namespace":"gloo-system"}}
```

**Comment by ashleywang1** [comment]
This happens when you have a version of Gloo installed that is lower than 1.5.0-beta8, where a RateLimitConfig CRD was introduced.

To fix this error, either upgrade Gloo to v1.5.0-beta10 or downgrade glooctl to the same version of Gloo that's running in your cluster.

**Comment by lgadban** [comment]
Hi @ashleywang1, the example I posted is when using both `glooctl` and Gloo OSS `1.5.0-beta10`
